samples/BuildStream/first-project/elements/hello.bst

```diff
@@ -0,0 +1,13 @@
+kind: import
+
+# Use a local source to stage our file
+sources:
+- kind: local
+  path: hello.world
+
+# Configure the import element
+config:
+
+  # Place the content staged by sources at the
+  # root of the output artifact
+  target: /
```

Comment:
Please add a sample from a real application under a permissive license. We prefer to avoid hello world & co. examples as they might bias the Bayesian classifier.